src/number.ts

```diff
@@ -13,3 +13,10 @@ export const toInt = (value: any, defaultValue: number = 0) => {
   const result = parseInt(value)
   return isNaN(result) ? defaultValue : result
 }
+
+export const round = (value: number, precision?: number): number => {
```

Comment:
I think we should default the `precision` arg to something common. If I call `round(2.239)` and I get back `2.239` that is probably unexpected and unhelpful in most cases. Defaulting this to a precision of `2` might make sense?

I could be wrong about this, what do you think?

Reply:
That's nice @rayepps  i will work on it.